src/solvers/dg/dg_2d.jl

```diff
@@ -366,6 +380,22 @@ function calc_volume_integral!(du::AbstractArray{<:Any,4}, u, nonconservative_te
   return nothing
 end
 
+# TODO: Taal dimension agnostic
+function calc_volume_integral!(du::AbstractArray{<:Any,4}, u, nonconservative_terms, equations,
+                               volume_integral::VolumeIntegralPureLGLFiniteVolume,
+                               dg::DGSEM, cache)
+  @unpack volume_flux_fv = volume_integral
+
+  # Loop over blended DG-FV elements
+  @timeit_debug timer() "pure FV" Threads.@threads for element in eachelement(dg, cache)
+    # Calculate LGL FV volume integral
```

Comment:
```suggestion
  # Calculate LGL FV volume integral
  @threaded for element in eachelement(dg, cache)
```
Same story as above.